### ISSUE TITLE
Extra closing brackets inserted based on number of buffers opened

### ISSUE THREAD

**Issue by mkrasnitski** [rstacruz/vim-closer#43]
When using neovim 0.8, extra closing brackets are inserted if more than one buffer has been opened. Killing buffers via `:bd` doesn't decrease the count, the count only ticks up when a buffer is created. This happens regardless of filetype (I tested with JS, C, and Rust).

Example (in rust):

Instead of this snippet:

```rust
fn test() {
```
being turned into this:

```rust
fn test() {

}
```

this gets inserted instead, when enter is pressed:

```rust
fn test() {

}
}
}
}
```

**Comment by dtrckd** [comment]
Hi,

I confirm the bug.
The vim version I use:  
VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Oct 01 2021 01:51:08)

Reproduce:  `vim -p a.go b.go`

```go
// edit a.go

func a() {
}
}
```

**Comment by dtrckd** [comment]
This PR #41 seems to solve that?.

Any chance it gets merged ?  @rstacruz

**Comment by mkrasnitski** [comment]
Closed by #42.
